pkg/bundle/setup/clustersetup.sh

```
@@ -213,34 +204,34 @@ EOF
-    pr_info  "patching default route"
+    pr_info "patching default route"
     oc patch -p '{"spec": {"host": "default-route-openshift-image-registry.'$EIP'.nip.io"}}' route default-route -n openshift-image-registry --type=merge
     stop_if_failed $? "failed patch default route"
     #sleep $STEPS_SLEEP_TIME
 }
 
 set_credentials() {
-    pr_info  "setting cluster credentials"
-    podman run --rm -ti ghcr.io/crc-org/crc-cloud/httpd-tools:latest developer $PASS_DEVELOPER > htpasswd.developer
+    pr_info "setting cluster credentials"
+    podman run --rm --network=none registry.access.redhat.com/ubi10/httpd-24 htpasswd -Bbn developer $PASS_DEVELOPER >htpasswd.developer
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)